Document/0x03-Overview.md

```diff
@@ -1,75 +1,86 @@
 
-From the view of a mobile app, this extra care has to be taken when storing user data, such as using appropriate key storage APIs and taking advantage of hardware-backed security features when available.
+When developing mobile apps, we must take extra care when storing user data. For example, we can use appropriate key storage APIs and take advantage of hardware-backed security features when available.
 
-On Android in particular, one has to deal with the problem of fragmentation. Not every Android device offers hardware-backed secure storage. Additionally, a large percentage of devices run outdated versions of Android with older API versions. If those versions are to be supported, apps must restrict themselves to older API versions that may lack important security features. When the choice is between better security and locking out a good percentage of the potential user base, odds are in favor of better security.
+Fragmentation is a problem we deal with especially on Android devices. Not every Android device offers hardware-backed secure storage, and many devices are running outdated versions of Android. For an app to be supported on these out-of-date devices, it would have to be created using an older version of Android's API which may lack important security features. For maximum security, the best choice is to create apps with the current API version even though that excludes some users.
 
 ### Communication with Trusted Endpoints
 
-Mobile devices regularly connect to a variety of networks, including public WiFi networks shared with other (possibly malicious) clients. This creates great opportunities for network-based attacks, from simple packet sniffing to creating a rogue access point and going SSL man-in-the-middle (or even old-school stuff like routing protocol injection - the bad guys aren't picky).
+Mobile devices regularly connect to a variety of networks, including public WiFi networks shared with other (possibly malicious) clients. This creates opportunities for network-based attacks that could be anything from simple packet sniffing (monitoring data passed over a network), to creating a rogue access point and an SSL man-in-the-middle attack (MITM eavesdrops on and alters communication between two parties), or even routing protocol injection. The bad guys aren't picky. For more information about MITM, see **Performing Man-in-the-Middle Attacks on the Network Layer** in the **Testing Network Communication** chapter.
```

Comment:
I actually removed the list entirely for now - I'm thinking the point of this section isn't to list possible attacks, it's so emphasize that we need to secure communication between apps and endpoints. What do you think?